packages/types/src/H256.ts

```diff
@@ -6,8 +6,12 @@ import { AnyU8a } from './types';
 
 import U8aFixed from './codec/U8aFixed';
 
-// Hash containing 256 bits (32 bytes), typically used in blocks, extrinsics and
-// as a sane default for fixed-length hash representations.
+/**
+ * @name Hash
```

Comment:
Fixed this one. I'm not 100% sure why name is needed - I've always assumed it is used when linking through, e.g. using `[[H256]]` somewhere will link through. And since it is basically used everywhere, just stuck with it.

Quite possibly it _may_ (completely untested) work without it. For now, just sticking it in everywhere, following the script.